core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml

```diff
@@ -1,6 +1,23 @@
 tide:
   queries:
-  - labels:
+  - includedBranches:
+    - main
+    labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - needs-rebase
+    repos:
+    - openshift/machine-api-provider-powervs
+  - includedBranches:
+    - openshift-4.11
+    - release-4.11
+    labels:
     - approved
     - lgtm
```

Comment:
add `bugzilla/valid-bug`  and `staff-eng-approved` here in line with other repos

Reply:
Added it